openapi-cli/src/test/resources/ballerina-to-openapi/advance/openapi/multiple_service_01.yaml

```diff
@@ -27,7 +27,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/ErrorPayload'
+                $ref: "#/components/schemas/ErrorPayload"
```

Comment:
What's the reason for this change? What will be the behaviour with the new parser version if the contract is having single quotes for the ref schema values?

Reply:
This change is only for the schema generation from the OAS model object. The parser works for both single and double quotes.

Reply:
I see. That'll be fine then